.github/workflows/python-package-conda.yml

```diff
@@ -0,0 +1,34 @@
+name: Python Package using Conda
+
+on: [push]
+
+jobs:
+  build-linux:
+    runs-on: ubuntu-latest
+    strategy:
+      max-parallel: 5
+
+    steps:
+    - uses: actions/checkout@v4
+    - name: Set up Python 3.10
+      uses: actions/setup-python@v3
+      with:
+        python-version: '3.10'
```

Comment:
The project currently runs on 3.13

https://github.com/duneanalytics/dune-client/blob/5177a1d4bd1ebd8e514495c52282dc032e50b8f7/.github/workflows/pull-request.yaml#L11